src/rpc/rawtransaction.cpp

```diff
@@ -2689,13 +2689,13 @@ static RPCHelpMan rawreissueasset()
             throw JSONRPCError(RPC_INVALID_PARAMETER, strprintf("Invalid asset address provided: %s", asset_address_uni.get_str()));
         }
 
-        size_t input_index = -1;
+        int input_index = -1;
         const UniValue& input_index_o = issuance_o["input_index"];
         if (input_index_o.isNum()) {
             input_index = input_index_o.get_int();
```

Comment:
no need to initialize `input index` above, can just `int input_index = input_index_o.get_int();`

Reply:
(why I'm commenting here at all? Just being curious what code ubsan complains about...)

Reply:
Lol! Good catch, if I need to touch this code again, I'll use your suggestion. (Or, once I get CI working, if I go back to clean things up..)